src/pysdmx/model/dataflow.py

```diff
@@ -395,6 +400,43 @@ def __str__(self) -> str:
         return ", ".join(out)
 
 
+class DataStructureDefinition(MaintainableArtefact, frozen=True):
+    """A collection of metadata concepts.
+
+    Attributes:
+        agency: The organization responsible for the data structure.
+        annotations: The list of annotations attached to the data structure.
+        id: The identifier of the data structure.
+        description: descriptive information about the data structure.
+        is_external_reference: Whether the data structure is an external
+            reference.
+        is_final: Whether the data structure is final.
+        name: The data structure's name.
+        service_url: The URL of the service providing the data structure.
+        structure_url: The URL of the structure.
+        uri: The URI of the data structure.
+        urn: The URN of the data structure.
+        valid_from: The date from which the data structure is valid.
+        valid_to: The date until which the data structure is valid.
+        version: The version of the data structure.
+    """
+
+    agency: Union[str, Agency] = ""
+    annotations: Sequence[Annotation] = ()
+    id: str = ""
+    description: Optional[str] = None
+    is_external_reference: bool = False
+    is_final: bool = False
+    name: Optional[str] = None
+    service_url: Optional[str] = None
+    structure_url: Optional[str] = None
+    uri: Optional[str] = None
+    urn: Optional[str] = None
+    valid_from: Optional[datetime] = None
+    valid_to: Optional[datetime] = None
+    version: str = "1.0"
```

Comment:
Out of curiosity, why are these fields defined again, although they are already defined in the parents (maintainable, versionable, etc.)?

Reply:
I put them all in the same place during development so that I don't have to navigate between inheritances. I forgot to remove them. Done.